packages/template/public/index.html

```diff
@@ -30,13 +28,13 @@
   <noscript>
     You need to enable JavaScript to run this app.
   </noscript>
-  <div id="interbit" data-boot-react-app="templatePublic" data-peer-hints="ib-dev----master.herokuapp.com" data-chain-id-template-public="8afdda40b86ad927fb5c52a0b9a19301217950ec721d3407dc902aed33750d20">
+  <div id="interbit" data-boot-react-app="templatePublic" data-peer-hints="" data-chain-id-template-public="88c712d781c185ad81675c0e700752288b9519be1d69771daeb68fdb5726435f">
```

Comment:
Revert this please. This chain ID does not match our deployed chains on our pipeline.